test/dlc_tests/container_tests/bin/efa/test_nccl_tests_using_efa

```diff
@@ -69,12 +34,24 @@ validate_all_reduce_performance_logs(){
 check_efa_nccl_all_reduce_performance(){
     benchmark=$(cat $TRAINING_LOG | grep '1073741824' | tail -n1 | awk -F " " '{{print $11}}' | sed 's/ //' | sed  's/  5e-07//')
     echo "Benchmark throughput: ${benchmark}"
+    if [[ -z "${benchmark}" ]]; then
+        echo "benchmark variable is empty"
+        exit 1
+    fi
 
     # The standard throughput should be at least 41 for 2 p4d with 4 EFA devices and 7 for 2 p3dn with 1 EFA device.
     # However, if the 2 instances are not in the same A-Z in the same region, performance can decrease.
     # To account for this we need to modify thresholds dynamically based on where instances are.
     # Temporarily setting these to be < 50% of optimal until AWS OFI NCCL team has concrete numbers for this.
-    PERFORMANCE_THRESHOLD="3"
+    if [[ ${INSTANCE_TYPE} == "p4d.24xlarge" ]]; then
+        PERFORMANCE_THRESHOLD="15"
+    elif [[ ${INSTANCE_TYPE} == "p4de.24xlarge" ]]; then
+        PERFORMANCE_THRESHOLD="18"
+    elif [[ ${INSTANCE_TYPE} == "p5.48xlarge" ]]; then
+        PERFORMANCE_THRESHOLD="230"
+    else
+        PERFORMANCE_THRESHOLD="3"
+    fi
```

Comment:
Could you please run this test on all of these instance types to show that these values are correct for PT 1.13 CUDA 11.7, PT 2.0 CUDA 11.8, and CUDA 12.1 DLCs?

Reply:
The TF EFA test is also meant to use these same scripts. We'd also need to make sure that the CUDA versions they use (starting from TF 2.10 DLCs) are compatible with these modified thresholds as well.

Reply:
will confirm logs, else will keep PERFORMANCE_THRESHOLD="3"